src/SAML2/Provider/HostedIdentityProviderProcessor.php

```diff
@@ -329,13 +373,18 @@ public function continueSingleSignOn()
     /**
      * @param Request $httpRequest
      * @return \Symfony\Component\HttpFoundation\Response
+     * @throws \AdactiveSas\Saml2BridgeBundle\Exception\InvalidArgumentException
      */
     public function processSingleLogoutService(Request $httpRequest)
     {
         $inputBinding = $this->bindingContainer->get($this->identityProvider->getSlsBinding());
 
         try {
-            $logoutMessage = $inputBinding->receiveSignedMessage($httpRequest);
+            if($this->identityProvider->wantSignedLogoutRequest()){
```

Comment:
First of all, I told myself that is the idp to require or not signing requests. But this is too strict, by let this being configured on SP, this is much more flexible